components/ide/jetbrains/gateway-plugin/src/main/kotlin/io/gitpod/jetbrains/gateway/GitpodStartWorkspaceView.kt

```diff
@@ -72,20 +73,14 @@ class GitpodStartWorkspaceView(
                     this.text = "https://github.com/gitpod-io/spring-petclinic"
                 }
             button("New Workspace") {
-                // TODO(ak) disable button if blank
                 if (contextUrl.component.text.isNotBlank()) {
-                    val backend = backendsModel.selectedItem
-                    val selectedBackendId = if (backend != null) {
-                        backendToId[backend]
-                    } else null
-                    val backendParam = if (selectedBackendId != null) {
-                        ":$selectedBackendId"
-                    } else {
-                        ""
+                    backendsModel.selectedItem?.let {
+                        backendToId[it]?.let { backend ->
+                            BrowserUtil.browse("https://${settings.gitpodHost}#referrer:jetbrains-gateway${backend}/${contextUrl.component.text}")
```

Comment:
But when trying to start the new workspace, it was failing, and I noticed is was due to a missing colon here:

```suggestion
                            BrowserUtil.browse("https://${settings.gitpodHost}#referrer:jetbrains-gateway:${backend}/${contextUrl.component.text}")
```

After fixing this, the PR will be good to be merged!